go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/Azure/go-autorest/autorest/azure/auth v0.1.0
 	github.com/Azure/go-autorest/autorest/to v0.2.0 // indirect
 	github.com/Azure/go-autorest/autorest/validation v0.1.0 // indirect
+	github.com/GoogleCloudPlatform/cloudsql-proxy v0.0.0-20190625211440-fa714b45b08a
```

Comment:
frustrating that their tagging strategy isn't go module compatible. Can you leave a comment of which tag you grabbed? If you didn't explicitly grab a tag, please `go get github.com/GoogleCloudPlatform/cloudsql-proxy/cmd/cloud_sql_proxy@1.14` and leave a comment.